pom.xml

```diff
@@ -43,6 +43,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-rest</artifactId>
 		</dependency>
+		<dependency>
+		    <groupId>org.springframework.data</groupId>
+		    <artifactId>spring-data-rest-hal-browser</artifactId>
+		</dependency>
```

Comment:
I'm not sure this should be enabled in production, but it gives us a really cool browser for exploring our API endpoints.

Reply:
This could be stuck in a maven profile and only activated on dev.